.github/workflows/docker_linux.yml

```diff
@@ -0,0 +1,58 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Docker-Linux
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+    paths:
+      - 'docker/**'
+
+  # Run builds for any PRs.
+  pull_request:
+    paths:
+      - 'docker/**'
```

Comment:
Should we change to docker/linux/**?

Reply:
I'm not very familiar with github action. Does github action support multiple workflows for testing repo, here docker_linux.yml and later docker_windows.yml?

Reply:
Yep it is expected that you break them out this way.